src/features/view-subdomains/SubdomainTableCard/SubdomainTableCard.tsx

```diff
@@ -60,13 +55,10 @@ export const SubdomainTableCard: FC<SubdomainTableCardProps> = ({
 		}
 	});
 
-	const metric = buyNowPrice?.price
-		? `${formatEthers(buyNowPrice?.price.toString())} ${paymentTokenSymbol}`
+	const metric = buyNowPrice
```

Comment:
Whilst we're here could we add `errorText: '-',` below line 96: 

<img width="1028" alt="Screenshot 2023-03-01 at 15 56 44" src="https://user-images.githubusercontent.com/39112648/222049343-db3883b1-ed58-4530-bd1e-3544468d7958.png">